framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/Commands/GenerateProxyCommand.cs

```diff
@@ -19,6 +19,7 @@ namespace Volo.Abp.Cli.Commands
     public class GenerateProxyCommand : IConsoleCommand, ITransientDependency
     {
         public static Dictionary<string, Dictionary<string, string>> propertyList = new Dictionary<string, Dictionary<string, string>>();
+        public static string output = "";
```

Comment:
Why created a static variable instead of a local one at the line below:
````csharp
output = commandLineArgs.Options.GetOrNull(Options.Output.Short, Options.Output.Long);
````

Also, `output = ""` value is always overwritten.

Reply:
Output value is used in more than one method. Instead of moving the variable between methods, I preferred to define it globally. I chose to use static considering that we might need to override the output value when writing the files.